package.json

```diff
@@ -89,13 +89,14 @@
     "jshint": "^2.9.3",
     "localstorage-memory": "^1.0.2",
     "mocha": "^3.0.2",
+    "mongodb": "^2.2.29",
```

Comment:
Used to generate an ObjectID from a string value. I probably could stub-out similar functionality, though.